src/settings.py

```diff
@@ -146,6 +146,7 @@ class GlobalSettings:
     # NSRT learning parameters
     min_data_for_nsrt = 0
     learn_side_predicates = False
+    sidelining_approach = "naive"
```

Comment:
I feel like we should just combine this with the `learn_side_predicates` setting. One setting is easier to keep track of than two! How about a single setting `side_predicate_learner`, with possible options: `no_learning`, `prediction_error_hill_climbing`, `preserve_skeletons_hill_climbing`?

Reply:
great idea!